lib/android_apk.rb

```diff
@@ -124,6 +124,7 @@ module Reason
   # @raise [AndroidApk::ApkFileNotFoundError] if the filepath doesn't exist
```

Comment:
The most of changes are produced by rubocop's autocorrect.

Extra changes are like

- a bug fix: https://github.com/DeployGate/android_apk/pull/33/files#r1000705352
- add some comments to disable rubocop